Makefile

```diff
@@ -0,0 +1,25 @@
+.PHONY: test test-verbose test-profile clean docker bootstrap
+
+PACKAGES = `go list ./... | grep -v vendor/`
+
+all: inertia
+
+inertia:
+	go build
+
+test:
+	go test $(PACKAGES) --cover
+
+test-verbose:
+	go test $(PACKAGES) -v --cover
+
+clean: inertia
+	rm -f inertia
+	rm -f cover.html
+
+docker:
```

Comment:
I thought we’d eventually make this an automated build, but this works for development

Reply:
Could change this to tag it as a test build in the future and add some `inertia [REMOTE] init` flags (ie `test-build`) to pull that build? Not sure if dockerhub supports that

Reply:
These two things happen at very different layers. One is when we’re tweaking daemon functionality, and one is when the user is trying to deploy the application. I suppose I could see creating a staging image for testing inertia that receives arbitrary pushes, but I’m not too in love with adding this type of functionality to the CLI...

Reply:
Actually I meant something closer to #35, where we implement something that keeps the daemon version on `[REMOTE] init` in sync with the local inertia, except everything that's not a proper tagged release we just call it `snapshot` maybe

So:
`inertia init -v 0.0.1` would download the docker build tagged as `v0.0.1`
`inertia init -v snapshot` would download the docker build tagged as `SNAPSHOT` which is dangerous and could be anything but I think that's the point of snapshot builds?